### PR TITLE
fix: latency visualize API new features in the Gallery section.

### PR DESCRIPTION
**Pull request by emori-ctc**
## Pre-review checklist for the PR author

- [ ] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [ ] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

**Review by rokamu623 — CHANGES_REQUESTED** [review]
The combination with the graph itself (ex: Callback frequency graph) should be added to the bulleted list.

Replace "Other..." with a sentence meaning "These elements can be visualized".

File: docs/gallery.md [hunk 1]
```diff
@@ -3,72 +3,62 @@
...
-#### Time-series graph
+- Callback frequency
+- Communication frequency
+- Pub/Sub frequency
+- Path frequency
+- Callback latency
+- Communication latency
+- Callback period
+- Communication period
+- Pub/Sub period
```
Comment (rokamu623) [inline docs/gallery.md:22]: Path response time

File: docs/gallery.md [hunk 2]
```diff
@@ -3,72 +3,62 @@
...
-#### Timestamp tables
-
-![callback_period_df](./imgs/callback_period_df.png)
+#### The following elements can be visualized
 
-#### Time-series graph
+- Callback frequency
+- Communication frequency
+- Pub/Sub frequency
+- Path frequency
```
Comment (rokamu623) [inline docs/gallery.md:17]: Path frequency is not implemented.
```suggestion
- Path frequency
```

File: docs/gallery.md [hunk 3]
```diff
@@ -3,72 +3,62 @@
...
+![callback_latency_histogram](./imgs/callback_latency_histogram.png)
 
-![callback_latency_df](./imgs/callback_latency_df.png)
+#### The following elements can be visualized
 
-#### Time-series graph
+- Callback frequency
+- Communication frequency
+- Pub/Sub frequency
+- Path frequency
```
Comment (rokamu623) [inline docs/gallery.md:35]: Path frequency is not implemented.
```suggestion
- Path frequency
```

File: docs/gallery.md [hunk 4]
```diff
@@ -3,72 +3,62 @@
...
+- Callback latency
+- Communication latency
+- Callback period
+- Communication period
+- Pub/Sub period
```
Comment (rokamu623) [inline docs/gallery.md:40]: Path response time

File: docs/gallery.md [hunk 5]
```diff
@@ -3,72 +3,62 @@
...
-
-### [Publish/Subscribe Frequency](./visualization/frequency/index.md#publish-and-subscription)
-
-![pub_sub_frequency_time_line](./imgs/pub_sub_frequency_time_line.png)
-
-### [Publish/Subscribe Period](./visualization/period/index.md#publish-and-subscription)
+![message_flow](./imgs/message_flow_sample.png)
 
-![pub_sub_period_time_line](./imgs/pub_sub_period_time_line.png)
+## [Callback Scheduling Visualization](./visualization/scheduling/callback.md)
```
Comment (rokamu623) [inline docs/gallery.md:54]: Application | Executor | Path | Node | CallbackGroup | Sequence[CallbackGroup]

**Review by rokamu623 — CHANGES_REQUESTED** [review]

File: docs/gallery.md [hunk 6]
```diff
@@ -3,72 +3,71 @@
...
+- Callback period
+- Communication period
+- Pub/Sub period
+- Path response time
 
-#### Time-series graph
+## Histogram
 
-![callback_period_time_line](./imgs/callback_period_time_line.png)
+### [callback latency graph](./visualization/latency/index.md)
```
Comment (rokamu623) [inline docs/gallery.md:26]:
```suggestion
### [Callback latency graph](./visualization/latency/index.md)
```

File: docs/gallery.md [hunk 7]
```diff
@@ -3,72 +3,71 @@
...
+## Stacked Bar
 
-![callback_latency_time_line](./imgs/callback_latency_time_line.png)
-
-### [Callback Scheduling Visualization](./visualization/scheduling/callback.md)
-
-#### Timing chart of callback scheduling
-
-![Callback_Scheduling_Visualization_sample](./visualization/imgs/callback_scheduling_executor.png)
+### [Response time graph](./visualization/path/response_time.md)
```
Comment (rokamu623) [inline docs/gallery.md:44]:
```suggestion
### [Path response time graph](./visualization/path/response_time.md)
```

**Review by rokamu623 — CHANGES_REQUESTED** [review]

**Review by rokamu623 — APPROVED** [review]
LGTM!

**Review by isp-uetsuki — APPROVED** [review]
LGTM

**Pull request merged**